### PR TITLE
Bullet Change 2: Blood Boogaloo.

### PR DESCRIPTION
**Pull request by Vizoplez**
## About the Pull Request
TM this PR or the other one. This WILL change gameplay massively, making hypovolemic shock a VERY fast killer.

## Why It's Good For The Game
El balance test.

## Changelog

:cl:
balance: Blood/Bleed changes, heart-rate affects bleeding, arterial bleed buffs, hit-chance changes + nerfs in some areas. This is experimental and will be subject to change.
/:cl:

### CODE REVIEW

File: code/modules/organs/external/_external.dm [hunk 1]
```diff
@@ -58,11 +58,11 @@
 
 	// Joint/state stuff.
 	var/joint = "joint"                // Descriptive string used in dislocation.
-	var/amputation_point               // Descriptive string used in amputation.
+	var/amputation_point               // Descriptive	string used in amputation.
```
Comment (Tsurupeta) [inline code/modules/organs/external/_external.dm]:
```suggestion
	var/amputation_point               // Descriptive string used in amputation.
```

File: code/modules/organs/internal/heart.dm [hunk 2]
```diff
@@ -226,4 +226,4 @@
 	. = "[pulsesound] pulse"
 
 /obj/item/organ/internal/heart/get_mechanical_assisted_descriptor()
-	return "pacemaker-assisted [name]"
\ No newline at end of file
+	return "pacemaker-assisted [name]"
```
Comment (Vizoplez) [inline code/modules/organs/internal/heart.dm:229]: tbh i didnt even change this.

Thanks git

File: code/game/machinery/vitals_monitor.dm [hunk 3]
```diff
@@ -163,7 +163,7 @@
 				if(beep)
 					playsound(src, 'sound/machines/ekg_alert.ogg', 40)
 				if(read_alerts)
-					alerts[PULSE_ALERT] = "Excessive heartbeat! Possible Shock Detected!"
+					alerts[PULSE_ALERT] = "Heart-rate exceeding 250BPM, Possible Shock Detected! Patient Expiry likely!"
```
Comment (Vizoplez) [inline code/game/machinery/vitals_monitor.dm:166]: These changes tell people a better idea on how not to kill their patient

File: code/modules/organs/internal/heart.dm (hunk 2 above)
Comment (Tsurupeta) [reply code/modules/organs/internal/heart.dm:229]: It's eof newline

**Comment by aidan-bowman** [comment]
At first glance, I'd say that the vitals monitor changes (i.e. the small descriptions of what's wrong with someone) are irrelevant to the bleeding changes, and as such should be pushed regardless of the rest of the PR (unless of course these descriptions only make sense with the new bleeding values).

**Review by Tsurupeta — APPROVED** [review]

File: code/game/machinery/vitals_monitor.dm [hunk 4]
```diff
@@ -197,7 +197,7 @@
 		else if(lungs.breath_fail_ratio < 1)
 			add_overlay(image(icon, icon_state = "breathing_shallow"))
 			if(read_alerts)
-				alerts[LUNGS_ALERT] = "Abnormal breathing detected!"
+				alerts[LUNGS_ALERT] = "Abnormality in patient breathing detected!"
```
Comment (Tsurupeta) [inline code/game/machinery/vitals_monitor.dm]:
```suggestion
				alerts[LUNGS_ALERT] = "Abnormality in patient's breathing detected!"
```

File: code/modules/mob/mob_helpers.dm [hunk 5]
```diff
@@ -98,33 +98,33 @@
...
+	BP_CHEST =  5, //This is CENTER OF MASS. You shouldn't have a super high flat-chance to miss. Base is 10
+	BP_GROIN =  25, //Killing someones liver is ultra-lethal. Make this slightly harder.
+	BP_L_ARM =  35, //Base is 25
+	BP_R_ARM =  35, // Ditto
+	BP_L_HAND = 50, //50% flat miss chance.
+	BP_R_HAND = 50, //Ditto
+	BP_L_LEG =  30, //Legs are pretty big, but they're almost always a moving target.
+	BP_R_LEG =  30, //Ditto
+	BP_L_FOOT = 45, //Feet are pretty big + close to the floor.
+	BP_R_FOOT = 45, //Ditto
```
Comment (Tsurupeta) [inline code/modules/mob/mob_helpers.dm]:
```suggestion
	BP_HEAD =   65,
	BP_CHEST =  5,
	BP_GROIN =  25,
	BP_L_ARM =  35,
	BP_R_ARM =  35,
	BP_L_HAND = 50,
	BP_R_HAND = 50,
	BP_L_LEG =  30,
	BP_R_LEG =  30,
	BP_L_FOOT = 45,
	BP_R_FOOT = 45,
```

File: code/modules/mob/mob_helpers.dm [hunk 6]
```diff
@@ -98,33 +98,33 @@
...
 
 //Used to weight organs when an organ is hit randomly (i.e. not a directed, aimed attack).
 //Also used to weight the protection value that armour provides for covering that body part when calculating protection from full-body effects.
 var/list/global/organ_rel_size = list(
-	BP_HEAD =   25,
-	BP_CHEST =  70,
-	BP_GROIN =  30,
+	BP_HEAD =   20, //Slightly harder to hit. Base 25
+	BP_CHEST =  85, //Really easy to hit, even unintentionally. Base 70
+	BP_GROIN =  40, //This is pretty big, should be higher.
```
Comment (Tsurupeta) [inline code/modules/mob/mob_helpers.dm]:
```suggestion
	BP_HEAD =   20,
	BP_CHEST =  85,
	BP_GROIN =  40,
```

File: code/modules/mob/mob_helpers.dm [hunk 7]
```diff
@@ -98,33 +98,33 @@
...
 	BP_L_ARM =  25,
 	BP_R_ARM =  25,
-	BP_L_HAND = 10,
-	BP_R_HAND = 10,
-	BP_L_LEG =  25,
+	BP_L_HAND = 5, // Base 10
+	BP_R_HAND = 5, //Hitting hands without a directed attack is hard. Base 10
+	BP_L_LEG =  25, //Legs, while a moving target, are pretty big. This stays the same.
```
Comment (Tsurupeta) [inline code/modules/mob/mob_helpers.dm]:
```suggestion
	BP_L_HAND = 5,
	BP_R_HAND = 5,
	BP_L_LEG =  25,
```

File: code/modules/mob/mob_helpers.dm [hunk 8]
```diff
@@ -98,33 +98,33 @@
...
 	BP_R_LEG =  25,
-	BP_L_FOOT = 10,
-	BP_R_FOOT = 10,
+	BP_L_FOOT = 5, //Ditto with hands.
+	BP_R_FOOT = 5,
```
Comment (Tsurupeta) [inline code/modules/mob/mob_helpers.dm]:
```suggestion
	BP_L_FOOT = 5,
	BP_R_FOOT = 5
```

**Review by Tsurupeta — CHANGES_REQUESTED** [review]

**Comment by aidan-bowman** [comment]
> balance: Blood + Hitchance.

DEFINITELY should make this more descriptive, holy shit.

**Comment by Vizoplez** [comment]
> At first glance, I'd say that the vitals monitor changes (i.e. the small descriptions of what's wrong with someone) are irrelevant to the bleeding changes, and as such should be pushed regardless of the rest of the PR (unless of course these descriptions only make sense with the new bleeding values).

They do. This changes how heartrate scales bleeding, and bleeding overall. These give pretty good IC warnings on how you should treat an injured individual (sedation/painkillers/lowering heartrate/repairing brain)

**Comment by aidan-bowman** [comment]
> > At first glance, I'd say that the vitals monitor changes (i.e. the small descriptions of what's wrong with someone) are irrelevant to the bleeding changes, and as such should be pushed regardless of the rest of the PR (unless of course these descriptions only make sense with the new bleeding values).
> 
> They do. This changes how heartrate scales bleeding, and bleeding overall. These give pretty good IC warnings on how you should treat an injured individual (sedation/painkillers/lowering heartrate/repairing brain)

Fair enough, although if the PR gets closed it would still be a good idea for a different PR changing just the messages. From the looks of it though, I doubt this is getting closed.

**Review by Tsurupeta — APPROVED** [review]

**Pull request merged**